OpenQA/Isotovideo/Utils.pm

```diff
@@ -143,7 +143,7 @@ sub handle_generated_assets {
     my ($command_handler, $clean_shutdown) = @_;
     my $return_code = 0;
     # mark hard disks for upload if test finished
-    return unless $command_handler->test_completed && ($bmwqemu::vars{BACKEND} eq 'qemu');
+    return if ($bmwqemu::vars{BACKEND} ne 'qemu');
```

Comment:
I would prefer if you keep the unless, i.e. `return unless $bmwqemu::vars{BACKEND} eq 'qemu';`

Reply:
fixed